crates/bevy_ecs/src/system/system_piping.rs

```diff
@@ -7,10 +7,11 @@ use crate::{
 };
 use std::borrow::Cow;
 
-/// A [`System`] that chains two systems together, creating a new system that routes the output of
-/// the first system into the input of the second system, yielding the output of the second system.
+/// A [`System`] created by piping the output of the first system into the input of the second.
 ///
-/// Given two systems `A` and `B`, A may be chained with `B` as `A.chain(B)` if the output type of `A` is
+/// This can be repeated indefintely, but system pipes cannot branch: the output is consumed by the receiving system.
```

Comment:
(beginner question, not a review question)

Can a hint be provided as to what could be done instead of piping to allow giving the input of one system to multiple systems? This seems like something that would come up a lot.


The answer might well be: "you should pipe into an adapter system that calls non-system functions to operate on the piped input"? Or perhaps there is another mechanism provided by Stageless, whose docs there should be a link to?

Reply:
Nothing, unless @DJMcNab has something to say about it. You would have to use events or some other data flow mechanism instead.

Reply:
(Ultimately this shouldn't need to be special cased at all, but that's well outside of the scope of this PR).